### PR TITLE
Add job status on expand row for hosts

### PR DESCRIPTION
**Pull request by nixocio**
Add job status on expand row for hosts

image

See: https://github.com/ansible/awx/issues/5236

### CODE REVIEW

File: awx/ui_next/src/screens/Host/HostList/HostListItem.jsx [hunk 1]
```diff
@@ -1,62 +1,99 @@
...
 
 function HostListItem({ host, isSelected, onSelect, detailUrl, rowIndex }) {
   const labelId = `check-action-${host.id}`;
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const {
+    summary_fields: { recent_jobs: recentJobs = [] },
+  } = host;
+
+  const recentPlaybookJobs = recentJobs.map(job => ({ ...job, type: 'job' }));
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Host/HostList/HostListItem.jsx:24]: Are we replacing the `type` value or adding it in to each object?

Comment (jakemcdermott) [inline awx/ui_next/src/screens/Host/HostList/HostListItem.jsx]: Are we replacing the type value or adding it in to each object?

Comment (nixocio) [reply awx/ui_next/src/screens/Host/HostList/HostListItem.jsx]: @jakemcdermott I was adding the type for every object. However this data is already present on the API endpoint.

image

**Review by jakemcdermott — APPROVED** [review]

**Comment by nixocio** [comment]
e2e test triggered.

**Review by tiagodread — APPROVED** [review]

**Pull request merged**